src/styles/product-card.css

```
@@ -0,0 +1,100 @@
+.product-card {
+  box-sizing: border-box;
+  max-width: 200px;
+  min-width: 80px;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: nowrap;
+  align-items: center;
+  justify-content: space-between;
+  /* gap: 40px; */
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+}
```

```suggestion
.product-card {
  width: 200px;
  align-items: center;
  padding: 32px 16px 16px;
  border: 1px solid #f3f3f3;
  border-radius: 5px;
  background-color: #fff;
}
```

Remove comments. Why you need flex? Block elements by default become one under another